tools/hangar/__snapshots__/test_corpus/valid/optionals.test.w_compile_tf-aws.md

```diff
@@ -127,7 +127,19 @@ class $Root extends $stdlib.std.Resource {
     $helpers.nodeof(this).root.$preflightTypesMap = { };
     let $preflightTypesMap = {};
     const cloud = $stdlib.cloud;
-    const Person = $stdlib.std.Struct._createJsonSchema({$id:"/Person",type:"object",properties:{age:{type:"number"},name:{type:"string"},},required:["age","name",]});
+    const Person = $stdlib.std.Struct._createJsonSchema({
+      $id: "/Person",
+      type: "object",
+      description: "",
+      properties: {
+        age: {  type: "number" , "description": "```wing\nage: num\n```" },
```

Comment:
The names of the props should be quotes in case they are not symbolic

```suggestion
        "age": {  type: "number" , "description": "```wing\nage: num\n```" },
```

Also think that "type" should be quoted just for consistency.